src/js/infoBlock.js

```diff
@@ -0,0 +1,28 @@
+const processor = require('.\\processor.js');
+const constants = require('.\\constants.js');
+
+class InfoBlock {
+
+  constructor(text) {
+    this.element = processor.createHtml('div', 'info-block-wrapper _hidden');
+    let infoBlock = processor.createHtml('div', 'info-block');
+    let infoBlockButton = processor.createHtml('div', 'info-block-button');
+
```

Comment:
we prefer to use constants for any 'string', or css class names too.
also u can check how css modules works. https://developerhandbook.com/webpack/how-to-configure-scss-modules-for-webpack/

scss modules + typescript are removing these gap. also modules helps in case of same names in the same project. e.g. u did class .loader for main page, and another team member did the same class for a list. That must be 2 different loader, and one piece of code will override another. In the end both of loader won't work as designed. Module approach gives unique name to each class by adding hash, generated on build, and it guarantees both of your .loader will be completely different classes.

Reply:
p.s. I don't need u setup modules in this project. Just read about them and try to use their advantages in your future projects

Reply:
Done